src/SFA.DAS.IdentifyDataLocks.Web/Helpers/InvokeWithExtension.cs

```diff
@@ -0,0 +1,13 @@
+﻿using SFA.DAS.IdentifyDataLocks.Domain;
+using System;
+
+namespace SFA.DAS.IdentifyDataLocks.Web.Pages
+{
+
+    public static class InvokeWithExtension
```

Comment:
This is hard to read without knowing the context :-)

Reply:
Yes, looking at it the following week I think you're right.  I have removed it.